### PR TITLE
better documentation for runST method, fixes #11

### LINKED ISSUE #11: what is the ST.run method? broken documentation?
here

https://github.com/purescript/purescript-foreign-object/blob/8a7dc21c48e3e6d31e253b0e58dd95a3bbd7f6e7/src/Foreign/Object.purs#L78

### PR DESCRIPTION
**Pull request by srghma**

### CODE REVIEW

**Comment by srghma** [comment]
@hdgarrood could you check, please)

**Comment by LiamGoodacre** [comment]
I don't know anything about the travis config, but other than that, this looks good.

**Comment by srghma** [comment]
Cool, thx)

File: src/Foreign/Object.purs [hunk 1]
```diff
@@ -75,7 +75,7 @@ freezeST :: forall a r. STObject r a -> ST r (Object a)
 freezeST = _copyST
 
 -- | Freeze a mutable Object, creating an immutable Object. Use this function as you would use
--- | `ST.run` to freeze a mutable reference.
+-- | `Control.Monad.ST.run` (from purescript-st package) to freeze a mutable reference.
```
Comment (hdgarrood) [inline src/Foreign/Object.purs]:
```suggestion
-- | `Control.Monad.ST.run` (from the `purescript-st` package) to freeze a mutable reference.
```

**Comment by thomashoneyman** [comment]
@srghma I've removed the Travis config file for you as we are no longer using Travis in the core libraries. Thanks!

**Comment by srghma** [comment]
updated

**Pull request merged**